Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build docker image at gcr.io/images-public/metadata:<tag>
+
+FROM golang:1.12
+
+ENV GO111MODULE on
+
+RUN apt-get update && apt-get -y install unzip patch && apt-get clean
+
+# Install Bazel
+RUN cd /tmp && \
+    wget -O /tmp/bazel-installer.sh https://github.com/bazelbuild/bazel/releases/download/0.24.1/bazel-0.24.1-installer-linux-x86_64.sh && \
+    chmod +x bazel-installer.sh && \
+    ./bazel-installer.sh --user
+
+ENV PATH=/root/bin:${PATH}
+
+WORKDIR /go/src/github.com/kubeflow/metadata
+
+COPY . .
+
+RUN /bin/bash api/generate_proto.sh
```

Comment:
This isn't needed, since we assume devs run this script whenever we update the API. For now, it may be sufficient to just build with `go build`.

Reply:
Generated code always concerns me. If we run this script before our presubmit tests, it slows down presubmit test every time. If we don't run this script for tests nor build, the proto and go code may be out of sync by mistake without notice.

What do you think?

Reply:
I think the presubmit test should check it, if  we want to make the gen code is in sync with the proto.

I removed this build step.